web-registry/src/components/organisms/RegistryLayout/RegistryLayout.tsx

```diff
@@ -17,59 +16,28 @@ import { useWallet } from 'lib/wallet/wallet';
 
 import { PageViewTracking } from 'components/molecules/PageViewTracking';
 
-import DefaultAvatar from '../../assets/avatar.png';
-import { chainId } from '../../lib/ledger';
-import { RegistryIconLink, RegistryNavLink, ScrollToTop } from '../atoms';
-import { AppFooter } from './AppFooter';
-import { WalletButton } from './WalletButton/WalletButton';
+import DefaultAvatar from '../../../assets/avatar.png';
+import { chainId } from '../../../lib/ledger';
+import { RegistryIconLink, RegistryNavLink, ScrollToTop } from '../../atoms';
+import { AppFooter } from '../AppFooter';
+import { WalletButton } from '../WalletButton/WalletButton';
+import {
+  getHeaderColors,
+  getIsTransparent,
+  getMenuItems,
+} from './RegistryLayout.config';
+import { fullWidthRegExp } from './RegistryLayout.constants';
```

Comment:
@flagrede It would even make sense to abstract the entire header, which invades the layout component, into an AppHeader component, so that we would have something more like:

```
  ...
  return (
    <>
      <AppHeader />
      <Outlet />
      <AppFooter />
      <PageViewTracking />
      <ScrollToTop />
      <CookiesBanner privacyUrl={URL_PRIVACY} TOSUrl={URL_TERMS_SERVICE} />
    </>
  );
```

Maybe we could even encapsulate `AppHeader` and `AppFooter` as subcomponents of `RegistryLayout`, maybe it doesn't make much sense to have them separate..
* `/RegistryLayout.AppFooter.tsx`
* `/RegistryLayout.AppHeader.tsx`

Reply:
Good idea will do!